src/main/java/uk/gov/ons/census/fwmt/rmadapter/service/impl/RMAdapterServiceImpl.java

```diff
@@ -31,9 +30,13 @@ public void sendJobRequest(ActionInstruction actionInstruction) throws GatewayEx
       gatewayEventManager
           .triggerEvent(actionInstruction.getActionRequest().getCaseId(), CANONICAL_CREATE_SENT, LocalTime.now());
     } else if (actionInstruction.getActionCancel() != null) {
-      jobServiceProducer.sendMessage(CanonicalJobHelper.newCancelJob(actionInstruction));
-      gatewayEventManager
-          .triggerEvent(actionInstruction.getActionCancel().getCaseId(), CANONICAL_CANCEL_SENT, LocalTime.now());
+      if (actionInstruction.getActionCancel().getAddressType().equals("HH")) {
+        jobServiceProducer.sendMessage(CanonicalJobHelper.newCancelJob(actionInstruction));
+        gatewayEventManager
+            .triggerEvent(actionInstruction.getActionCancel().getCaseId(), CANONICAL_CANCEL_SENT, LocalTime.now());
+      } else {
+        gatewayEventManager.triggerEvent(actionInstruction.getActionCancel().getCaseId(), CANONICAL_CANCEL_FAILED, LocalTime.now());
```

Comment:
ah i see you are sending error messages when your address type isn't HH. i don't think this is required. May be able to just ignore and dead letter the message instead?

Reply:
Part of the ticket wants an error log to be sent if the case is not HH.

Reply:
Okay 👍